docs/why-dbos.md

```diff
@@ -44,56 +62,45 @@ If your program is ever interrupted or crashes, DBOS uses this saved state to re
 For example, if your checkout workflow crashes right after validating payment, instead of the order being lost forever, DBOS recovers from a checkpoint and goes on to ship the order.
 Thus, DBOS makes your application **resilient to any failure**.
 
-### DBOS Is Lightweight
+## DBOS vs. Other Systems
 
-All you need to use DBOS is install the open-source DBOS Transact library ([Python](https://github.com/dbos-inc/dbos-transact-py), [TypeScript](https://github.com/dbos-inc/dbos-transact-ts)).
+### DBOS vs. Temporal
 
-To add DBOS to your application, simply annotate _workflows_ and _steps_ in your program like this:
-<Tabs groupId="language">
-<TabItem value="python" label="Python">
+Both DBOS and Temporal provide durable execution, but DBOS is implemented in a lightweight Postgres-backed library whereas Temporal, and architecturally similar systems such as Restate and Inngest, are implemented in an externally orchestrated server.
 
-```python
-@DBOS.step()
-def step_one():
-    ...
+You can add DBOS to your program by installing the open-source library, connecting it to Postgres, and annotating workflows and steps.
+By contrast, to add Temporal to your program, you must rearchitect your program to move your durable workflows to a Temporal worker, configure a Temporal server to orchestrate those workflows, and access your workflows only through a Temporal client.
```

Comment:
```suggestion
By contrast, to add Temporal to your program, you must rearchitect your program to move your durable workflows and tasks (activities) to separate Temporal workers, configure a Temporal server to orchestrate those workflows, and access your workflows only through a Temporal client.
```

Reply:
I think there are three components: a workflow server, a worker server for activities, and a Temporal server.